net-mail/mu/metadata.xml

```diff
@@ -15,6 +15,7 @@
 	</longdescription>
 	<use>
 		<flag name="emacs">Install the mu4e mail client.</flag>
+		<flag name="mug">Install the mug toy client, not for normal use.</flag>
```

Comment:
Please don't add dot here, it's just a one-line description instead of a sentence. (Also remove from emacs while at it)